### PR TITLE
Enhanced Documentation, Fix "stop after x messages"

### PR DESCRIPTION
**Pull request by SimonButtazzi**
Added some words about configuration and output examples

### CODE REVIEW

**Comment by SimonButtazzi** [comment]
Fixed "stop after" x messages (pre_stop state).
State didn't work for stop after > 1.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "node-red-contrib-power-monitor",
-    "version": "1.2.0",
+    "version": "1.2.3",
```
Comment (xoseperez) [inline package.json:3]: Set it to 1.2.1

**Review by xoseperez — CHANGES_REQUESTED** [review]
Since this is a minor change and bug fix, version should ber 1.2.1

**Comment by SimonButtazzi** [comment]
> Since this is a minor change and bug fix, version should be 1.2.1
You are right, initially I added docs and fixed the bug in a new version.
-> Changed version

File: package.json (hunk 1 above)
Comment (SimonButtazzi) [reply package.json]: Hey, can you please merge?
Fix was committed, see last comment.

Thanks, Simon

**Review by xoseperez — APPROVED** [review]

**Comment by xoseperez** [comment]
Merged and updated on npmjs.

**Pull request merged**